src/hooks/voucherHooks.ts

```diff
@@ -35,12 +35,7 @@ export const useVoucher = (): {
   }, []);
 
   useEffect(() => {
-    if (isConnected && isMainserverUp && !isLoggedIn) {
-      logInCallback();
-    } else {
-      setIsLoading(false);
-      setIsError(true);
-    }
+    logInCallback();
```

Comment:
isn't `setIsLoading(false);`  missing now for some cases?

Reply:
There is `setIsLoading(false)` statement in `loginCallback`. Therefore, I do not think it is missing.